src/pytest_cov/engine.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, cov_source, cov_report, cov_config, cov_append, config=None,
         """Get some common config used by multiple derived classes."""
         self.cov_source = cov_source
         self.cov_report = cov_report
-        self.cov_config = os.path.abspath(cov_config)
+        self.cov_config = os.path.abspath(cov_config) if os.path.exists(cov_config) else cov_config
```

Comment:
Creating the combiner instance early along with the other one would also make this issue go away. It does make me wonder if this just really isn't a use-case that combine was intended for.